packages/core/src/components/dropdown/index.js

```diff
@@ -330,8 +344,6 @@ class Dropdown {
   };
 
   onClick = e => {
-    e.stopPropagation();
-    e.preventDefault();
```

Comment:
why are these removed?

Reply:
this is removed because click handler is listening entire body to handle "click outside" and this is removed that it would not interfere with other click listeners. For example form submit button e.t.c Found this issue while integrating Dropdown into Landlords form

Reply:
ah ok